### PR TITLE
Fix some automation stuff

### PR DESCRIPTION
**Pull request by armstrongnate**
[ignore-commit-lint]

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: TestsFoundation/TestsFoundation/Element/LoginFindSchool.swift [hunk 1]
```diff
@@ -30,4 +30,8 @@ public enum LoginFindAccountResult {
     public static func item(host: String) -> Element {
         return app.find(id: "LoginFindAccountResult.\(host)")
     }
+
+    public static func item(label: String) -> Element {
+        return app.find(label: label)
+    }
```
Comment (cobbal) [inline TestsFoundation/TestsFoundation/Element/LoginFindSchool.swift]: Does this need to be a separate function from `app.find`?

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1007392&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.45% | 93.43% | 0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: TestsFoundation/TestsFoundation/Element/LoginFindSchool.swift (hunk 1 above)
Comment (armstrongnate) [reply TestsFoundation/TestsFoundation/Element/LoginFindSchool.swift]: Nope. Fine by me.

**Pull request merged**